Nighttime-Operations/Auxiliary-Telescope/AT-Pre/Prepare-for-On-Sky-Operations.rst

```diff
@@ -0,0 +1,112 @@
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: **
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: **
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AT-Pre-Open-for-On-Sky-Operations:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#############################
+Prepare for On-Sky Operations
+#############################
+
+.. _Open-for-On-Sky-Operations-Overview:
+
+Overview
+========
+
+This procedure gets the AuxTel system ready for on-sky operations, preparing the telescope and dome to be fully open. 
+
+In a standard night, it will follow the execution of the venting (link) procedure, but it works from any initial state of the system. 
+
+The observers must prepare for on-sky between 2.5 hours and 1 hour before sunset. 
+
+.. _Open-for-On-Sky-Operations-Prerequisites:
+
+Prerequisites
+=============
+
+- Before taking the decision to open, review the weather conditions and :ref:`weather constraints <Nighttime-Operations-Weather-Constraints>` page.
+
+- AuxTel is fully ready to operate and all components are enabled. 
+
+- The :ref:`daytime checkout <AT-Daytime-Checkout-Daytime-Checkout>` has been executed successfully. 
+
+- The telescope has been venting (link to venting once merged) for a considerable amount of time. 
+  If venting wasn't possible, this procedure will take the system from the :ref:`calibration <statesLATISS-Combined-Calibrations-Procedure-LATISS-Combined-Calibrations-Generation-Procedure>` 
+  state or any other initial state to be prepared for on-sky observations. 
+
+.. _Open-for-On-Sky-Operations-Post-Condition:
+
+Post-Condition
+==============
+
+- Telescope is ready for on-sky operations. 
+  
+  Around 15 minutes before the end of the evening nautical twilight, proceed with the :ref:`beginning of the night procedure to center, align and focus the telescope. <AT-On-sky-WEP>`
+
+.. _Open-for-On-Sky-Operations-Procedure-Steps:
+
+Procedure Steps
+===============
+#. Review procedure :ref:`prerequisites <Open-for-On-Sky-Operations-Prerequisites>` and confirm they have been executed. 
+
+#. Announce in the #summit-announce and #summit-auxtel Slack channel that AuxTel is preparing to go on-sky. 
+
+#. Command the telescope and dome to prepare for on-sky observations. 
```

Comment:
I would like to see a brief comment (maybe within a note box) saying that this script can be queued if the vent script is already running, to say explicitly that people don't need to wait to run it. It is obvious, but not for people that is just learning and/or seeing this for the first time.

Reply:
👍